### ISSUE TITLE
Example traces on the homepage not available

### ISSUE THREAD

**Issue by amosjyng** [oughtinc/ice#222]
Hello!

When I try to click on "Placebo classification and description" on the homepage, the JS tries to access [`https://oughtinc.github.io/static/traces/01GCZNZ1YC0XRE1QHSAV6MPWJD/trace.jsonl`](https://oughtinc.github.io/static/traces/01GCZNZ1YC0XRE1QHSAV6MPWJD/trace.jsonl), which 404's. I assume that this is not intentional? :)

**Comment by alexmojaki** [comment]
Those trace files are still in an old, uncompatible format, e.g. the trace for that file is stored at https://oughtinc.github.io/static/traces/01GCZNZ1YC0XRE1QHSAV6MPWJD/trace.jsonl. We've been meaning to rerun the recipes so that ice.ought.org can be updated with new code and viewing e.g. https://ice.ought.org/traces/01GCZNZ1YC0XRE1QHSAV6MPWJD includes the latest features, but we've been lagging. I hadn't really thought about this opposite but equally broken perspective. Maybe we should just hide all those traces in the meantime.

**Comment by amosjyng** [comment]
Haha no worries, though just FYI nothing seems to exist at https://oughtinc.github.io/ , and on top of that the CORS request from `ice.ought.org` fails :P

**Comment by lslunis** [comment]
Yeah, right now we're just using `https://oughtinc.github.io` for static file serving

**Comment by amosjyng** [comment]
Right, I'm just saying, it looks like static files are not being served from `https://oughtinc.github.io` as the site apparently doesn't currently exist

**Comment by alexmojaki** [comment]
I edited the first sentence of my comment which had said the wrong thing.

There is definitely a file at https://oughtinc.github.io/static/traces/01GCZNZ1YC0XRE1QHSAV6MPWJD.jsonl that exists and is being served, which is what we need. But there's no index page at https://oughtinc.github.io/.

>  the CORS request from ice.ought.org fails

I don't know what you mean. https://ice.ought.org/traces/01GCZNZ1YC0XRE1QHSAV6MPWJD looks fine to me and I see no network or console errors.

**Comment by amosjyng** [comment]
Ohhh my bad, I got that confused with the other trace URL you posted. All good now!
